### DISCUSSION TITLE
Two / Multiple currency support (for bank transfers)?

### DISCUSSION THREAD

**Discussion by c0de-bender** [pretix/pretix#3683] [Q&A] [answered]
Hi, is it somehow possible to have 2 currencies configured, so that prices can be shown and paid in those 2 currencies?

We don't necessarily need a complete multiple currency support, but we want to use bank transfer and it would be great if the order confirmation page would show the price in both currencies, so that we can tell people which amount of money in the "secondary" currency they have to transfer.

We don't necessarily need (semi-)automatic recognition of 2nd currency payments from a bank statement either. We are fine with manually marking those orders as paid. But people should know how much to transfer.

If there's no proper support for that, but a workaround, that would also help!

Background: We want to use Pretix for a festival hosted in Poland, but a lot of people from different parts of Europe will participate. Thus we want to offer payments in EUR and PLN. When using a payment service provider, it might take care of the conversion during the payment process, but we still want to give people the choice how to pay and offer bank transfer. Unfortunately, with only one currency there's no way to display the correct amount for the transfer in the second currency.

Note: I assume that Pretix doesn't have full support for 2 currencies. I've seen that Pretix includes the option to show prices in 2 different currencies in invoices, but I didn't find anything about displaying in the web interface itself.

Is there a plan to support two currencies in the future, or is this out of reach (e.g. because technically too complex)?

Thanks for any help!

**Comment by raphaelm** [comment, marked as answer]
It is out of reach currently. The level of support you require could be done in a plugin, but I don't think we want to do it in a core plugin.

**Comment by c0de-bender** [comment]
That's sad to hear. Thanks for the quick feedback!
